browser/profiles/profile_util_unittest.cc

```diff
@@ -78,3 +82,44 @@ TEST_F(BraveProfileUtilTest,
   brave::SetDefaultSearchVersion(GetProfile(), true);
   ASSERT_EQ(GetPrefs()->GetInteger(prefs::kBraveDefaultSearchVersion), 1);
 }
+
+// SetWebTorrentEnabled
+// ####################
+
+// No entry yet. Check initialized value
+TEST_F(BraveProfileUtilTest, SetWebTorrentEnabledExistingProfileNoEntryYet) {
+  const PrefService::Preference* pref =
+      GetPrefs()->FindPreference(kWebTorrentEnabled);
+  EXPECT_TRUE(pref->IsDefaultValue());
```

Comment:
why not using `HasUserSetting()` here also?

Reply:
Fixed!